pyproject.toml

```
@@ -24,13 +24,13 @@ Flask-Cors = "3.0.9"
 Flask-JWT-Extended = "3.25.0"
 Flask-Migrate = "2.1.1"
 Flask-SQLAlchemy = "2.5.1"
-raven = {extras = ["flask"], version = "^6.10.0"}
 psycopg2 = "2.9.3"
 simplejson = "3.12.0"
 fints = "3.0.1"
 gevent = "20.12.1"
 gunicorn = "20.0.4"
 MarkupSafe = "2.0.1"
+sentry-sdk = {extras = ["flask"], version = "^1.5.5"}
```

Then, we let dependabot make a pull request for every new upgrades. If it's green we merge, if not we change our code for any deprecated usage of the lib.
```suggestion
sentry-sdk = {extras = ["flask"], version = "*"}
```